lib/cred_stash/secret.rb

```diff
@@ -10,8 +10,8 @@ def initialize(name:, value: nil, key: nil, encrypted_value: nil, hmac: nil, con
     @context = context
   end
 
-  def encrypt!
-    @key = CredStash::CipherKey.generate(context: @context)
+  def encrypt!(kms_key_id: nil)
```

Comment:
Similar here. `nil` propagates down to `CredStash::CipherKey`, which will default to `alias/credstash` correctly.